libs/application/template-api-modules/src/lib/modules/templates/estate/utils/mappers.ts

```diff
@@ -35,6 +36,8 @@ const estateMemberMapper = (element: EstateMember) => {
           phone: '',
           email: '',
         }
+      : info(element?.nationalId).age < 18
```

Comment:
For reference on the kennitala package in its current state.
`info(undefined).age < 18` is evaluated to `undefined < 18` → `false`